src/main/java/gift/application/member/dto/MemberModel.java

```diff
@@ -22,4 +26,11 @@ public record IdAndJwt(
 
     }
 
+    public record InfoAndJwt(
```

Comment:
해당 부분은 토큰 중에서 Jwt라는 구현 상세를 나타내어 지나치게 결합되어 있는 것 같아요!
해당 부분에 대해서는 추상적으로 네이밍해두어도 좋을 것 같네요ㅎㅎ
```suggestion
    public record InfoAndToken(
```

Reply:
넵 참고하겠습니다!